cmd/priv_val_server/main.go

```diff
@@ -37,7 +37,7 @@ func main() {
 		*chainID,
 		*addr,
 		pv,
-		crypto.GenPrivKeyEd25519(),
+		ed25519.GenPrivKeyEd25519(),
```

Comment:
suggestion: now that this is prefixed with a proper package name (`ed25519` instead of `crypto`) don't you think this should be named `ed25519.GenPrivKey()` instead?

Reply:
Agreed